cirq/contrib/qasm_import/_parser.py

```diff
@@ -35,7 +36,42 @@ def __init__(self, supported_format: bool, qelib1_include: bool,
         self.circuit = c
 
 
-class QasmParser(object):
+class QasmGate:
```

Comment:
Don't call this a `Gate`; that has a very specific meaning in cirq that this class does not satisfy. Instead call it... maybe a statement? An invocation?

Reply:
Sounds good! I like `QasmGateStatement`, will go with that.